src/service/ampdoc-impl.js

```diff
@@ -300,6 +303,14 @@ export class AmpDoc {
   contains(node) {
     return this.getRootNode().contains(node);
   }
+
+  /**
+   * Binding of 3p analytics vendors' responses, used amp-ad-exit
+   * @return {!Object<string, !Object<string, Object<string,string>>>}
+   */
+  getIframeTransportResponses() {
```

Comment:
(If anyone wants to replicate this, it was done with https://github.com/ampproject/amphtml/pull/11061 at change #f53dece)